mlpf/tfmodel/datasets/CMSDatasetFactory.py

```diff
@@ -16,7 +16,7 @@ def get_dataset(self, dataset_name, dataset_dict, split, max_examples_per_split=
             as_supervised=False,
             data_dir=dataset_dict["data_dir"],
             with_info=True,
-            shuffle_files=False,
+            shuffle_files=True,
```

Comment:
I'm not certain about this. This means every time we initialize the dataset, the test/train events will be different and plots will be not easily reproducible.